### PR TITLE
LIN-1969: add testimonial view

### PR DESCRIPTION
**Pull request by myeyesareblind**
Screenshot 2024-05-14 at 11 10 55 AM

### CODE REVIEW

File: RevenueCatUI/Views/TestimonialsView.swift [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+struct TestimonialsView_Previews: PreviewProvider {
+
+    static var previews: some View {
+        TestimonialsView()
+    }
+
+}
```
Comment (g-laures) [inline RevenueCatUI/Views/TestimonialsView.swift]: With macros you can now simply write 
```swift
#Preview {
    TestimonialsView()
}
```

File: RevenueCatUI/Views/TestimonialsView.swift [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            Text(text)
+                .foregroundStyle(Color(red: 32 / 255.0, green: 32 / 255.0, blue: 32 / 255.0))
+                .font(.system(size: 13, weight: .regular))
+            HStack {
+                Image(.nastyaCropped)
+                    .resizable(resizingMode: .stretch)
+                    .background(content: {
+                        Rectangle().fill(Color(red: 214 / 255.0, green: 207 / 255.0, blue: 204 / 255.0))
+                    })
+                    .frame(width: 40, height: 40)
```
Comment (g-laures) [inline RevenueCatUI/Views/TestimonialsView.swift]: Expected size is 42

**Review by g-laures — CHANGES_REQUESTED** [review]
Looks good in LM but DM is not supported:
Capture d’écran 2024-05-15 à 11 59 46

File: RevenueCatUI/Views/TestimonialsView.swift (hunk 2 above)
Comment (myeyesareblind) [reply RevenueCatUI/Views/TestimonialsView.swift]: indeed 👍

File: RevenueCatUI/Views/TestimonialsView.swift (hunk 1 above)
Comment (myeyesareblind) [reply RevenueCatUI/Views/TestimonialsView.swift]: I'm having problems with it due to iOS13/15 etc, do you have a sample/can you share?

**Comment by myeyesareblind** [comment]
> Looks good in LM but DM is not supported:

yeah the design was only added yesterday, AFAIK.

File: RevenueCatUI/Views/TestimonialsView.swift (hunk 1 above)
Comment (g-laures) [reply RevenueCatUI/Views/TestimonialsView.swift]: I only removed the code from line 52 and kept the `@available` and it works:
```swift
#if DEBUG

@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
@available(macOS, unavailable)
@available(tvOS, unavailable)
#Preview {
    TestimonialsView()
}

#endif
```

**Review by g-laures — APPROVED** [review]

**Pull request merged**